src/mypy.ini

```diff
@@ -1,7 +1,65 @@
 [mypy]
 strict_optional = True
 no_implicit_optional = True
-ignore_missing_imports = True
 warn_incomplete_stub = True
 check_untyped_defs = True
 show_error_context = True
+
+[mypy-rados]
```

Comment:
do we want `.*` at the end (basically everywhere) so we catch subpackages, too?

Reply:
the rados cython module doesn't have sub packages.